app/models/access_token.rb

```diff
@@ -0,0 +1,19 @@
+class AccessToken < ApplicationRecord
+  validates :token, :owner, presence: true
+
+  attr_accessor :users_token
+
+  scope :active, -> { where(deactivated_at: nil) }
+
+  before_validation :generate_users_token, :set_token
+
+private
+
+  def generate_users_token
+    self.users_token = SecureRandom.uuid if users_token.blank?
+  end
+
+  def set_token
+    self.token = Digest::SHA256.hexdigest(users_token)
```

Comment:
Yeah, we can change it but because we have already released this to AWS, we would either need to drop the db in AWS or create a migration. I'll make this change as a separate PR.